src/winter_onboarding_2021/fleet_management_service/utils.clj

```diff
@@ -21,3 +32,36 @@
     {:flash {:error true
              :style-class "alert alert-danger"
              :message msg}}))
+
+;;namespace 
+(defn remove-namespace [coll]
+  (postwalk
+   (fn [key]
+     (if (keyword? key)
+       (keyword (name key))
+       key))
+   coll))
+
+(defn recursicve-ns [ns hashmap map-keys]
```

Comment:
Can you explain to me how this function works?

Reply:
It take two arguments - 
1. An unnamespaced hashmap
2. Vector containing keys of the hashmap passed as first argument 

It is a recursive function, which checks for the second argument (keys vector). If it is:
- not nil, it will call itself with the first key in the vector dissociated and the same key associated with a namespace.
- nil - It will return the current hashmap

Reply:
Modified the function to make it more readable and obvious.